packages/puppeteer-core/src/util/disposable.ts

```diff
@@ -326,11 +332,11 @@ export class AsyncDisposableStack {
       throw errors[0];
     } else if (errors.length > 1) {
       let suppressed = null;
-      for (const error of errors.reverse()) {
+      for (const error of errors) {
```

Comment:
The second reverse is also wrong.
We dispose the resources in reverse order 
So A,B  will get disposed  in order B -> A, and if both throw  Error from B should be suppressed by error from A.